notebooks/file_calibration.ipynb

```diff
@@ -773,6 +779,12 @@
     "          1451: (224044799.8046875, 2.6075219,  5.9),\n",
     "          1452: (224166870.1171875, 2.6075219, 22.6),\n",
     "          1510: (231246948.2421875, 0.1068141, 23.9)}\n",
+    "\n",
+    "if hd.freqs[0] != 46920776.3671875 or hd.Nfreqs != 1536:\n",
+    "    # We have less frequencies than usual (maybe testing)\n",
+    "    phs_sol = {np.argmin(np.abs(hd.freqs - freq)): (freq, heading, chisq) for chan, (freq, heading, chisq) in phs_sol.items() if hd.freqs[0] <= freq <= hd.freqs[-1]}\n",
+    "\n",
+    "\n",
```

Comment:
Can this be avoided? I really don't like the hardcoding for frequencies and Nfreqs.

Reply:
Yeah, I cringed when I wrote this. I think the problem is that the `phs_sol` dict as originally defined already assumes these things implicitly (i.e. it assumes we have files whose first channel is 46.9 MHz and that have 1536 freqs). I could change the way that `phs_sol` is defined, and we could define each RFI channel by its frequency instead of its channel number. However, that would take a little longer to compute, and usually we don't need to worry about it.

Reply:
Ok, I guess I can live with this. Can we make make `hd.freqs[0] != 46920776.3671875` something more like `np.abs(hd.freqs[0] - 46920776.3671875) < 1e-3` so it's less vulnerable to round-off error?